mayastor/src/bdev/nexus/nexus_child_error_store.rs

```diff
@@ -48,6 +51,11 @@ pub struct NexusErrStore {
     records: Vec<NexusChildErrorRecord>,
 }
 
+pub enum NexusErrStoreQuery {
+    Total,
```

Comment:
please document the members and the purpose

Reply:
Enum purpose and values now described in comments.